biodatacatalyst-ui/src/main/webapp/psamaui/overrides/fenceLogin.js

```diff
@@ -40,6 +40,10 @@ define(['psamaSettings/settings', 'jquery', 'handlebars', 'text!login/fence_logi
                     var currentSession = JSON.parse(sessionStorage.getItem("session"));
                     currentSession.queryTemplate = queryTemplateResponse[0].queryTemplate;
                     currentSession.queryScopes = meResponse[0].queryScopes;
+                    currentSession.privileges = meResponse[0].privileges;
+                    if (currentSession.privileges.length > 1) {
+                        currentSession.privileges.push("FENCE_AUTHORIZED_ACCESS");
```

Comment:
Does anyone have objections to adding this fake (front-end-only) privilege? Originally I wanted to create a new session attribute called "authorizedAccess" or something of that nature. However, that would have led to the awkward situation of the header in the base hpds-ui having to add the whole session to the header view for no apparent reason.